src/Credentials/ClientCredentials.php

```diff
@@ -40,7 +42,32 @@ public function __construct($identifier, $secret, $callbackUri = null)
     }
 
     /**
-     * {@inheritdoc}
+     * Attempts to create client credentials from given options.
+     *
+     * @param  array   $options
+     *
+     * @return ClientCredentials
+     * @throws League\OAuth1\Client\Exceptions\ConfigurationException
+     */
+    public static function createFromOptions(array $options)
+    {
+        array_map(function ($required) use ($options) {
+            if (!array_key_exists($required, $options)) {
+                ConfigurationException::handleMissingRequiredOption($required);
```

Comment:
This points to a bit of a code smell... The reason you have to have `@codeCoverageIngore` is because these exceptions are throwing themselves. If you changed the signature to:

```php
throw ConfigurationException::missingRequiredOption($required);
```

The coverage would work properly.